test_helpers/main.go

```diff
@@ -32,12 +35,28 @@ type StartOpts struct {
 	// https://www.tarantool.io/en/doc/latest/reference/configuration/#cfg-basic-listen
 	Listen string
 
+	// PingServer changes a host to connect to test startup of a Tarantool
+	// instance. By default, it uses Listen value as the host for the connection.
+	PingServer string
+
+	// PingTransport changes Opts.Transport for a connection that checks startup
+	// of a Tarantool instance.
+	PingTransport string
+
+	// PingSsl changes Opts.Ssl for a connection that checks startup of
+	// a Tarantool instance.
+	PingSsl tarantool.SslOpts
```

Comment:
Thank you. It's a good idea to replace `Ping` because it's a implementation detail. So I've renamed `PingServer`, `PingTransport`, `PingSsl` to `ClientServer`, `ClientTransport`, `ClientSsl`.

It useful to set these values directly in tests (to set the wrong ones, for example, we use SSL for a server, but not for a client).